tests/test_yask.py

```diff
@@ -117,10 +117,10 @@ def test_fixed_halo_w_ofs(self, space_order):
             1 4 4 ... 4 1
             1 1 1 ... 1 1
         """
-        grid = Grid(shape=(16, 16, 16))
+        grid = Grid(shape=(17, 17, 17))
```

Comment:
why?

Reply:
Don't lile even grid shapes, makes weird (irrational like .999999999999) values for spacing.